addons/medical_treatment/functions/fnc_treatment.sqf

```diff
@@ -179,6 +179,19 @@ if (isArray (_config >> "sounds")) then {
     ];
 };
 
+private _startCallback = getText (_config >> "callbackStart");
+if (isNil _startCallback) then {
```

Comment:
Yes it can. To be honest this is just copied from the code that calls `callbackFailure`.
I can be Nil if its not a function or variable name. In that case its compiled. It got confused first as well.